### PR TITLE
[IMP] mail: add message backlink on replyingto in composer

### PR DESCRIPTION
**Pull request by rapr-odoo**
**Current behavior before PR:**

There is no backlink to message on 'Replying To' in header of Composer

**Desired behavior after PR is merged:**

There is a backlink to message

Task-2664839

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mail/static/src/models/composer_view.js [hunk 1]
```diff
@@ -182,6 +182,16 @@ registerModel({
             ev.preventDefault();
             this.discard();
         },
+        /**
+         * @private
+         * @param {MouseEvent} ev
+         */
+        onClickReplyingToMessage(ev) {
```
Comment (seb-odoo) [inline addons/mail/static/src/models/composer_view.js:189]: alphabetical order for methods

File: addons/mail/static/src/models/composer_view.js [hunk 2]
```diff
@@ -182,6 +182,16 @@ registerModel({
...
+            if (!this.threadView || !this.threadView.replyingToMessageView) {
+                return;
+            }
```
Comment (seb-odoo) [inline addons/mail/static/src/models/composer_view.js:192]: Did you have the issue actually happen with either of those being undefined or is it just preventive?

While in theory they are justified, we typically don't guard click handlers unless the issue actually happens.

Comment (rapr-odoo) [reply addons/mail/static/src/models/composer_view.js:192]: Ya. It was just for preventive manner. Although, it is not creating any issues right now. So, I have removed it right now.

**Review by seb-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**